### PR TITLE
Convert Yosys binary strings to the format specified from the device database.

### PR DESCRIPTION
**Pull request by litghost**
Peer changes are https://github.com/Xilinx/RapidWright/pull/141 and https://github.com/SymbiFlow/fpga-interchange-schema/pull/19

### CODE REVIEW

File: fpga_interchange/yosys_json.py [hunk 1]
```diff
@@ -103,7 +103,31 @@ def make_gnd_cell(cell, module_data, consts):
...
+def convert_parameters(device, cell, cell_type, property_map):
+    """ Convert cell parameters to match expression type from default. """
+    for name in property_map.keys():
+        definition = device.get_parameter_definition(cell_type, name)
+        if definition is None:
+            # This parameter doesn't have a special definition, don't touch it.
+            continue
+
+        if not definition.is_integer_like():
+            # Non-integer like parameters come from yosys as a string, leave
```
Comment (gatecat) [inline fpga_interchange/yosys_json.py:115]: You will need to trim a space off the end if one has been added to disambiguate a bitvector (iirc trim a space off the end if it matches `[01xz]* +`)

Comment (litghost) [reply fpga_interchange/yosys_json.py:115]: Python already strips:
```
int(' 01101010101010 ', 2)
```

So I think this is okay.

Comment (gatecat) [reply fpga_interchange/yosys_json.py:115]: But this is for the non-integer-like path - the space(s) are added to disambiguate what is intended to be a string

Comment (litghost) [reply fpga_interchange/yosys_json.py:115]: Ok.  Let me update.

Comment (litghost) [reply fpga_interchange/yosys_json.py:131]: I think this is fixed, take a look.

Comment (gatecat) [reply fpga_interchange/yosys_json.py:131]: LGTM

**Pull request merged**